server/filters/formatPersonalInterestsFilter.test.ts

```diff
@@ -0,0 +1,27 @@
+import formatPersonalInterestsFilter from './formatPersonalInterestsFilter'
+
+describe('formatPersonalInterestsFilter', () => {
+  describe('should format view model values', () => {
+    Array.of(
+      { source: 'COMMUNITY', expected: 'Community' },
+      { source: 'CRAFTS', expected: 'Crafts' },
+      { source: 'CREATIVE', expected: 'Creative' },
+      { source: 'DIGITAL', expected: 'Digital' },
+      { source: 'KNOWLEDGE_BASED', expected: 'Knowledge-based' },
+      { source: 'MUSICAL', expected: 'Musical' },
+      { source: 'OUTDOOR', expected: 'Outdoor' },
+      { source: 'NATURE_AND_ANIMALS', expected: 'Nature and animals' },
+      { source: 'SOCIAL', expected: 'Social' },
+      { source: 'SOLO_ACTIVITIES', expected: 'Solo activities' },
+      { source: 'SOLO_SPORTS', expected: 'Solo sports' },
+      { source: 'TEAM_SPORTS', expected: 'Team sports' },
+      { source: 'WELLNESS', expected: 'Wellness' },
+      { source: 'OTHER', expected: 'Other' },
+      { source: 'NONE', expected: 'None of these' },
```

Comment:
Minor, but this will need to be simply "None" in all your filters and associated tests

"None of these" is from the CIAG UI and is in context of a drop down where the user is making a selection.
When we display the value in our app it is in context of re-playing what the user entered, so "None of these" makes no sense (if I saw "none of these", I would say "none of what?")